controller/array_action/array_mediator_interface.py

```diff
@@ -210,12 +196,13 @@ def unmap_volume(self, volume_id, host_name):
         raise NotImplementedError
 
     @abstractmethod
-    def get_snapshot(self, snapshot_name, pool_id=None):
+    def get_snapshot(self, volume_id, snapshot_name, pool=None):
         """
         This function return snapshot info about the snapshot.
         Args:
-            snapshot_name : name of the snapshot in the storage system
-            pool_id: pool_id of the volume to find the snapshot more efficiently.
+            volume_id : id of the source volume (used to get pool in case pool parameter not given).
+            snapshot_name : name of the snapshot in the storage system.
```

Comment:
if you think the new convention should be to end every parameter description with a dot, then please align it everywhere accordingly, otherwise please keep it as is

Reply:
there is no consistency in the interface so that is the current convention.. I'm just keeping it.. in `unmap_volume` it's the same as this..

Reply:
I agree it's not consistent, but most of the args descriptions don't end with a dot, and specifically this line, so I don't see a reason to only add a dot here, unless a full alignment is being done.

Reply:
done